it/mail/chrome/messenger-smime/msgSecurityInfo.properties

```diff
@@ -27,43 +27,43 @@ SIHeaderMismatch=L'indirizzo e-mail del firmatario indicato nel certificato è d
+EIValid=Questo messaggio è stato crittografato prima di essere inviato. La crittografia rende molto difficile per altre persone leggere le informazioni in transito attraverso la rete.
 
 EIInvalidLabel=Il messaggio non può essere decifrato.
-EIInvalidHeader=Questo messaggio è stato cifrato prima di essere inviato, ma non può essere decifrato.
+EIInvalidHeader=Questo messaggio è stato crittografato prima di essere inviato, ma non può essere decifrato.
 
 EIContentAltered=Il contenuto del messaggio sembra essere stato alterato durante la trasmissione.
-EINoDecryptCert=The certificate used to encrypt the message cannot be found.
-EIPasswordError=You did not enter your Master Password correctly.
-EIInvalidCipher=The message was encrypted using an encryption strength that this version of your software does not support.
-EIClueless=Si sono verificati problemi sconosciuti con questo messaggio cifrato.
+EINoDecryptCert=Il certificato utilizzato per crittografare il messaggio non può essere trovato.
+EIPasswordError=Non hai inserito correttamente la tua Password principale.
+EIInvalidCipher=Il messaggio è stato crittografato utilizzando una crittografia la cui forza non è supportata da questa versione del tuo software.
+EIClueless=Si sono verificati problemi sconosciuti con questo messaggio crittografato.
 
 ## Signing Power Information string
-SPCanLabel=Message Can Be Signed
-SPCanHeader=You have chosen to digitally sign this message before sending it.
-SPCan=When other people receive your signed message, they can verify that the message comes from you and that it has not been altered since you signed it.
+SPCanLabel=Il messaggio può essere firmato
+SPCanHeader=Hai scelto di firmare digitalmente questo messaggio prima d'inviarlo.
+SPCan=Quando altre persone ricevono il tuo messaggio firmato, possono verificare che il messaggio proviene da te e che non è stato alterato da quando lo hai firmato.
 
-SPCannotLabel=Message Cannot Be Signed
-SPCannotHeader=You cannot digitally sign this message.
+SPCannotLabel=Il messaggio non può essere firmato
+SPCannotHeader=Non puoi firmare digitalmente questo messaggio.
 
-SPNoCert=You have not specified a valid certificate for creating digital signatures.
-SPNoSigCert=The certificate you have specified for signing messages cannot be used for that purpose.
-SPNoValidCert=The certificate you have specified for signing messages is not yet valid. Make sure your computer's clock is set correctly.
-SPExpiredCert=The certificate you have specified for signing messages has expired.
-SPRevokedCert=The certificate you have specified for signing messages has been revoked.
+SPNoCert=Non hai specificato un certificato valido per la creazione di firme digitali.
+SPNoSigCert=Il certificato che hai specificato per firmare i messaggi non può essere utilizzato per quello scopo.
+SPNoValidCert=Il certificato che hai specificato per firmare i messaggi non è valido. Assicurati che l'orologio del tuo computer è impostato correttamente.
```

Comment:
CORRECTIONS NEEDED. 

In the second sentence (beginning with "Assicurati") replace "è" with "sia".